khelo/lib/ui/flow/leaderboard/leaderboard_screen.dart

```diff
@@ -0,0 +1,231 @@
+  }
+
+  Widget _tabView(BuildContext context) {
+    final tabs = LeaderboardField.values;
+
+    return SingleChildScrollView(
+      padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 9),
+      scrollDirection: Axis.horizontal,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: List.generate(
+          tabs.length,
+          (index) => Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 4),
+            child: TabButton(
+              tabs[index].getString(context),
+              onTap: () => _controller.jumpToPage(index),
+              selected: index == _selectedTab,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _content(BuildContext context, LeaderboardViewState state) {
+    return Expanded(
+      child: PageView(
+        controller: _controller,
+        onPageChanged: notifier.onTabChange,
+        children: LeaderboardField.values
+            .map((field) => _playerListView(context, state, field))
+            .toList(),
+      ),
+    );
+  }
+
+  Widget _playerListView(BuildContext context, LeaderboardViewState state,
+      LeaderboardField field) {
+    final players = state.leaderboard
+            .firstWhereOrNull((data) => data.type == field)
+            ?.players ??
+        [];
+
+    if (players.isEmpty) {
+      return _emptyLeaderboard(context);
+    }
+    return ListView.separated(
+      padding: EdgeInsets.all(16),
+      itemBuilder: (context, index) {
+        final player = players[index];
+        return _leaderboardCell(
+          player: player,
+          field: field,
+          rank: index + 1,
+        );
+      },
+      separatorBuilder: (context, index) => SizedBox(height: 16),
+      itemCount: players.length,
+    );
+  }
+
+  Widget _leaderboardCell({
+    required LeaderboardPlayer player,
+    required LeaderboardField field,
+    required int rank,
+  }) {
+    return Container(
+      padding: EdgeInsets.all(16),
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(12),
+        border: Border.all(color: context.colorScheme.outline),
+      ),
+      child: Row(
+        children: [
+          Text(
+            rank.toString().padLeft(2, '0'),
+            style: AppTextStyle.subtitle2
+                .copyWith(color: context.colorScheme.textPrimary),
+          ),
+          SizedBox(width: 16),
+          ImageAvatar(
+            initial: player.user?.nameInitial ?? "?",
+            imageUrl: player.user?.profile_img_url,
+            size: 42,
+          ),
+          SizedBox(width: 16),
+          Expanded(
+            child: Text(
+              player.user?.name ?? "Deactivated User",
+              style: AppTextStyle.subtitle2
+                  .copyWith(color: context.colorScheme.textPrimary),
+            ),
+          ),
+          SizedBox(width: 8),
+          Material(
+            type: MaterialType.transparency,
+            child: Chip(
```

Comment:
If you don't want to make clickable then i think you need to use normat container instead of `Chip`.